docs/_data/portland-2022-sessions.yaml

```diff
@@ -314,51 +314,6 @@
     </ul>
 
     <p>If you need to create something from nothing, this talk is for you.</p>'
-- title: Inclusive Language
```

Comment:
Already did this in the announcement PR. Woops.

Reply:
Yeah, I noticed - no worries